### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Home_Focus Visible: Keyboard focus is not clearly visible in all the four sides for the "IntelliSense", "Run and Debug", "Built-in Git", "Extensions" which are present in the home screen.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#6199]
### GitHub Tags

#WCAG2.4.7; #A11ySev3; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Focus Visible; #Keyboard; #A11yAuto; #TabStops;

### Environment Details

Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 111.0.5563.147 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Verify whether the keyboard focus is clearly visible in all the four sides for the "IntelliSense", "Run and Debug", "Built-in Git", "Extensions" which are present in the home screen or not.

### Actual Result
Keyboard focus is not clearly visible in all the four sides for the "IntelliSense", "Run and Debug", "Built-in Git", "Extensions" which are present in the home screen.

### Expected Result
Keyboard focus should be clearly visible in all the four sides for the "IntelliSense", "Run and Debug", "Built-in Git", "Extensions" which are present in the home screen.

### User Impact
Users will find difficulty to identify where exactly the keyboard focus is present in the section.

### Attachment
![Bug_Focus](https://user-images.githubusercontent.com/99243079/229982636-c99e4177-c424-4e4b-bce8-40c1671960ee.gif)

**Comment by rzhao271** [comment]
Hi, this bug has been resolved.

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
URL:https://code.visualstudio.com/
The issue is fixed. Hence closing the bug.
Please find the attachment for the reference.
#Regressed:05-10-23; #Closed;
![Bug_Fixed](https://github.com/microsoft/vscode-docs/assets/99243079/5bd30ea4-121e-4e70-84a5-9890e4f53e66)
